_layouts/default.html

```diff
@@ -36,3 +36,4 @@
   {% include script.html %}
 </body>
 </html>
+{% include comments.html %}
```

Comment:
Ich glaube du möchtest den `include` an anderer Stelle, nämlich in der `post.html` setzen. 

Setzt du den hier include hier, kannst du theoretisch auf jeder Seite kommentieren, wenn du `comments: true` in dat yaml frontmatter schreibst.
Falls du das möchtest, solltest du den include in Zeile 28 verschieben (vor das schließende `main` element)